warehouse/query-core/src/main/java/datawave/query/tables/ScannerFactory.java

```diff
@@ -539,13 +540,29 @@ public void applyConfigs(ScannerBase scannerBase, String tableName) {
     public void applyConfigs(ScannerBase scannerBase, String hintKey, String tableName) {
 
         if (consistencyLevelMap != null && !consistencyLevelMap.isEmpty()) {
-            String key = consistencyLevelMap.containsKey(hintKey) ? hintKey : tableName;
-            scannerBase.setConsistencyLevel(consistencyLevelMap.get(key));
+            ScannerBase.ConsistencyLevel level = consistencyLevelMap.get(hintKey);
+            if (level == null) {
+                level = consistencyLevelMap.get(tableName);
+            }
+
+            if (level == null) {
+                log.trace("no consistency level found for table: {} key: {}", tableName, hintKey);
+            } else {
+                scannerBase.setConsistencyLevel(level);
+            }
         }
 
         if (executionHintMap != null && !executionHintMap.isEmpty()) {
-            String key = executionHintMap.containsKey(hintKey) ? hintKey : tableName;
-            scannerBase.setExecutionHints(executionHintMap.get(key));
+            Map<String,String> hint = executionHintMap.get(hintKey);
+            if (hint == null) {
```

Comment:
Is it possible to combine the log trace statement in this first clause? Same question would apply for the other if statements below.

Reply:
I think those log statements should stay separate for clarity. It's quite possible to have consistently levels configured correctly, but the scan hint is incorrect and vice versa.